app/src/main/java/dto/LocationAdapter.kt

```diff
@@ -30,6 +30,4 @@ class LocationAdapter : RecyclerView.Adapter<CustomViewHolder>() {
     }
 }
 
-class CustomViewHolder(val view: View) : RecyclerView.ViewHolder(view) {
-
-}
+class CustomViewHolder(val view: View) : RecyclerView.ViewHolder(view)
```

Comment:
Redundant empty class body.

Reply:
Is the class needed?  If there is a RecyclerView, it likely is.  The ViewHolder will need to be implemented, if so,  Instead of removing the empty block, I'd just finish out the concept.